LIMITATIONS.md

```diff
@@ -0,0 +1,67 @@
+# Security limitations
+
+This document's goal is to provide a short discussion on what security
+properties access to /dev/sev-guest has, specifically in a Cloud setting.
+
+## Initial assumptions
+
+For claims in this document, we assume that the VM firmware is vendored and
+signed by the vendor, or not signed at all. This will be the common use case.
+
+If a virtual machine monitor gives you access to provide your own launch image
+and signed IDBlock, then you have full control and responsibility over what that
+image does, and how careful it is to give workloads access to the SEV
+device. Since this library is for wrapping the Linux driver /dev/sev-guest which
+will be booted from a UEFI firmware, further discussion of other uses of SEV-SNP
+are out of scope.
+
+## Implications of a vendored firmware
+
+Your Cloud service provider may launch SEV-SNP VMs with their own build of UEFI
+firmware. This build will be used everywhere, partial rollouts notwithstanding.
+Because the firmware is the same everywhere, the measurement in the attestation
+report will be the same everywhere. The IDBlock, if provided, will also be the
+same everywhere.
+
+The security of an attestation measurement or a derived key are proportional to
+the specificity of the initially measured image. If the initial image is
+available to everyone and can run any workload, then keys bound to its
+measurement are known to everyone.
```

Comment:
When I read the spec, I parse it the same way as you've written here. However, this is not what DavidK said when I spoke w/ him last week. Let me send him an email to clarify this before we submit this wording. I just wan to make sure what we're writing here is correct.

Reply:
Added the extra bits about IDBlock behavior.